lib/teaspoon/suite.rb

```diff
@@ -94,8 +94,10 @@ def asset_from_file(original)
       normalize_js_extension(filename)
     end
 
-    def normalize_js_extension(filename)
-      filename.gsub(".erb", "").gsub(/(\.js\.coffee|\.coffee|\.es6|\.js\.es6|\.es6\.js)$/, ".js")
+    def normalize_js_extension(original_filename)
```

Comment:
@jejacks0n I'll look tomorrow but do you know offhand if sprockets can do this normalization for us? Would be nice.